packages/fxa-settings/src/components/InputText/index.tsx

```diff
@@ -132,7 +134,10 @@ export const InputText = ({
         <input
           className={classNames(
             inputOnlyClassName,
-            'pb-1 pt-5 px-3 w-full font-body rounded text-start',
+            'pb-1 pt-5 px-3 font-body rounded text-start',
+            // password specific width is to accomodate password manager icons
+            // that are otherwise hidden and inaccessible under the visibility icon
+            isPasswordInput ? 'w-[90%]' : 'w-full',
```

Comment:
I see the ticket says it's Safari specific. Should we use a similar hack as we did with `input-text` (see the tailwind file with `SAFARI HACK` note) or change that approach to yours?

Reply:
Thanks for pointing this out, I hadn't seen that we had a hack for this in Backbone! I would keep them separate though since implementation of the input and visibility toggle components is pretty different in Backbone vs React. The previously used hack isn't as simple to implement with tailwind, nested label and visibility button that uses absolute positioning. Perhaps we can revisit in https://mozilla-hub.atlassian.net/browse/FXA-7896